bitcoin/src/lib.rs

```diff
@@ -60,6 +60,9 @@ pub extern crate base64;
 /// Bitcoin base58 encoding and decoding.
 pub extern crate base58;
 
+#[cfg(feature = "bitcoinconsensus")]
+extern crate actual_bitcoinconsensus as bitcoinconsensus;
```

Comment:
In b461457c50ec7aaf9866b267c8c607edefa5dcbc:

Could you also pull the "move `bitcoinconsensus` into an explicit feature" change into its own commit? And clarify in the commit message that we need to do this because we need the feature to turn on the same feature in `units`, and we can't do any re-export tricks since this dependency is never re-exported for stability reasons.

Reply:
I started having a go at this but I don't really like it, it means adding a patch that introduces an unused feature because the `bitcoinconsensus` dependency is added in this work.

Reply:
Lq9000905@gmail.com